.github/workflows/nightly_teleport_operations_procedure_stable.yml

```diff
@@ -0,0 +1,93 @@
+    workflow_dispatch:
+        inputs:
+            helm-versions:
+                description: The Helm versions to use as comma separated list
+                type: string
+    pull_request:
+        branches-ignore:
+            - stable/**
+        # For now limit automatic execution to a minimum, can always be done manually via workflow_dispatch for a branch
+        paths:
+            - .github/workflows/nightly_teleport_operational_procedure_stable.yml
+            - .github/workflows/reuseable_teleport_operational_procedure.yml
+            - aws/dual-region/kubernetes/**
+            - test/**
+
+# limit to a single execution per ref (branch) of this workflow
+concurrency:
+    group: ${{ github.workflow }}-${{ github.ref }}
+    cancel-in-progress: true
+
+env:
+    AWS_PROFILE: infex
+    TESTS_TF_BINARY_NAME: terraform
+
+jobs:
+    dynamic-matrix:
+        runs-on: ubuntu-latest
+        outputs:
+            matrix: ${{ steps.generate-matrix.outputs.matrix }}
+        steps:
+            - name: Generate matrix
+              id: generate-matrix
+              run: |
+                  # renovate: datasource=helm depName=camunda-platform registryUrl=https://helm.camunda.io versioning=regex:^11(\.(?<minor>\d+))?(\.(?<patch>\d+))?$
+                  c86=11.1.1
+
+                  # Snapshot is used for namespace setup to do the DNS chaining tests with a static namespace. It's not used for the matrix.
+                  # shellcheck disable=SC2034
+                  c87=SNAPSHOT
+
+                  if [ "${{ inputs.helm-versions }}" == "" ]; then
+                    versions='{"helmChartVersion":["'${c86}'"]}'
+                  else
+                  output=$(echo "${{ inputs.helm-versions }}" | awk -F, '{
+                      for(i=1;i<=NF;i++) {
+                        if ($i ~ /^".*"$/) {
+                          printf("%s%s", $i, (i==NF)?"":",")
+                        } else {
+                          printf("\"%s\"%s", $i, (i==NF)?"":",")
+                        }
+                      }
+                    }')
+                    versions='{"helmChartVersion":['${output}']}'
+                  fi
+
+                  echo "${versions}"
+                  echo "matrix=${versions}" >> "$GITHUB_OUTPUT"
+
+    operational-procedure:
+        needs:
+            - dynamic-matrix
+        strategy:
+            # parallelism is limited due to infra cluster resource constraints.
+            # Once multiple versions are compatible with testing through teleport, investigate if increasing is possible.
+            max-parallel: 1
+            fail-fast: false
+            matrix:
+                helmChartVersion: ${{fromJson(needs.dynamic-matrix.outputs.matrix).helmChartVersion}}
+        uses: ./.github/workflows/reusable_teleport_operational_procedure.yml
+        with:
+            helmChartVersion: ${{ matrix.helmChartVersion }}
+            awsProfile: infex
+            testsTfBinaryName: terraform
```

Comment:
same question as in the snapshot workflow about global env vs here hardcoded again